.github/workflows/publish/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+bazel build //deploy/helm/scf:chart
+built_file="$(realpath bazel-bin/deploy/helm/scf/scf-*.tgz)"
```

Comment:
I think technically it should involve `$(bazel info bazel-genfiles)` instead of `bazel-bin`, but yeah it's kind of annoying that bazel doesn't give you a good way to get the outputs.

Alternatively we can do `bazel build //deploy/helm/scf:chart --build_event_json_file=…` and then try to parse that file.

Reply:
`bazel-bin` vs `bazel-genfiles` is just a matter of setting it up on the helm rules that we created. I particularly like to use `bazel-genfiles` only for transient generated stuff.

For parsing the output, I'll explore more the build event protocol.

Reply:
The JSON output is bugous. We would need to consume the protobuf output, then this gets a bit out of scope from this PR. I'll create a card for this.